Packages/Sources/ConnectWalletFeature/ConnectWalletFeature.swift

```diff
@@ -207,50 +207,51 @@ struct PopUpModifier: ViewModifier {
           .transition(.opacity.animation(.easeInOut))
           .zIndex(1)
 
-        VStack {
+        VStack(spacing: 0) {
           Text("This page will open another application.", bundle: .module)
             .multilineTextAlignment(.center)
             .foregroundStyle(Color.neutral2)
             .font(Font(FontName.poppinsRegular, size: 12))
+            .frame(width: 215)
```

Comment:
This line can be removed. The width of the text should be driven by the VStack width, which is driven by padding 32 (this is fine as you have it)